### ISSUE TITLE
what's the difference of Sparse Prediction and Dense Prediction?

### ISSUE THREAD

**Issue by huanghanmei** [AlexeyAB/darknet#5543]
I have a question about the yolo v4 paper: I have not a  clear understanding of the difference of Sparse Prediction and Dense Prediction？

**Comment by WongKinYiu** [comment]
dense prediction make prediction of every location of an image.
while sparse prediction only make prediction of some sampled region (region proposal) of an image.
![image](https://user-images.githubusercontent.com/12152972/81465038-7be96900-91f9-11ea-999c-fcedb5b236e5.png)

**Comment by huanghanmei** [comment]
Thank you. And can you give me a clear definition of Self-Adversarial Training, CmBN, and CSP?  I am not understand about this.

**Comment by AlexeyAB** [comment]
* Self-Adversarial Training - self-criticism, prevents overfitting, it allows network to resist adversarial attacks and to train 100 times more iterations without overfitting

* CmBN - collects statistics across samples in batch (instead of mini-batch as in Batch-normalization), it allows you to train by using GPU with small GPU-RAM size

* CSP - paththrough connections, (1) it allows some part of data to bypass a multiple conv3x3 layers if higher receptive field isn't required for this part of data, and (2) it brings spatial data to the end of network (in a similar way as the Image Pyramid does)
